### PR TITLE
PTX-22998 [TC Automation] AddNewDiskToKubevirtVM 

### PR DESCRIPTION
**Pull request by spai-px**
**What this PR does / why we need it**:
This PR Adds AddNewDiskToKubevirtVM test case 
**Which issue(s) this PR fixes** (optional)
Closes # PTX-22998 

**Special notes for your reviewer**:
[Jenkins run:](https://jenkins.pwx.dev.purestorage.com/job/Users/job/krishna/job/tp-pxe-byoc/2116/)
[Aetos run:](https://aetos.pwx.purestorage.com/resultSet/testSetID/592569 )

### CODE REVIEW

File: tests/basic/kubevirt_test.go [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+	. "github.com/onsi/ginkgo/v2"
+	"github.com/portworx/sched-ops/task"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/portworx/torpedo/pkg/log"
+	"time"
+
+	. "github.com/portworx/torpedo/tests"
+)
+
+var _ = Describe("{AddNewDiskToKubevirtVM}", func() {
```
Comment (dbhatnagar-px) [inline tests/basic/kubevirt_test.go:14]: @spai-px - looks like we will be using these methods a lot as I can see few test cases based on this. Can we have this test case as a common function itself which basically, counts current disks, attaches "n" number of new disks and then recounts the total disks again.

Comment (spai-px) [reply tests/basic/kubevirt_test.go:10]: Yeah this seems like we can move it to a function. I have created a function AddDisksToKubevirtVM and created a new file in tests folder named as kubevirt_helper.go.

Comment (spai-px) [reply tests/basic/kubevirt_test.go:11]: As discussed have moved the implementation to drivers folder. under kubevirt folder

**Review by dbhatnagar-px — APPROVED** [review]
Lgtm

**Pull request merged**